src/Microsoft.Azure.ServiceBus/Microsoft.Azure.ServiceBus.csproj

```diff
@@ -2,7 +2,8 @@
 
   <PropertyGroup>
     <Description>This is the next generation Azure Service Bus .NET Standard client library that focuses on queues &amp; topics. For more information about Service Bus, see https://azure.microsoft.com/en-us/services/service-bus/</Description>
-    <VersionPrefix>3.0.0</VersionPrefix>
+    <VersionPrefix>3.0.1</VersionPrefix>
```

Comment:
Is this needed? `Verstion` is already set.

Reply:
Honestly, the whole thing is quite confusing. I'm trying to automate few systems. Some work with `<VersionPrefix>`, and some work with `<Version>`.. So I'm adding both of them.
 Do you see any concerns?

Reply:
I'll try to remove the `Prefix` version and see if I can change the automation to work without it.